include/seqan3/alphabet/container/concatenated_sequences.hpp

```diff
@@ -556,7 +556,9 @@ class concatenated_sequences
     reference operator[](size_type const i)
     {
         assert(i < size());
+        SEQAN3_WORKAROUND_GCC_BOGUS_MEMCPY_START(-Warray-bounds)
```

Comment:
This one is new due to failure on gcc-git

https://cdash.seqan.de/viewBuildError.php?buildid=316061

<details><summary>Click to show screenshot. Nightly builds are only kept for 3 days</summary>

![cdash seqan de_viewBuildError php_buildid=316061](https://github.com/user-attachments/assets/63bdf48f-d92a-4bb9-abed-0ddfed8a269a)

</details>